### PR TITLE
New package: newm-0.3alpha

### PR DESCRIPTION
**Pull request by Zoh-j02r**
#### Testing the changes
- I tested the changes in this PR: **YES**

#### New package
- This new package conforms to the [package requirements](https://github.com/void-linux/void-packages/blob/master/CONTRIBUTING.md#package-requirements): **YES**

### CODE REVIEW

**Comment by paper42** [comment]
alpha software using unstable wlroots will not be accepted

**Comment by Zoh-j02r** [comment]
pushing back newm to version 0.2 and using wlroots 0.15.1 will solve the issue?

**Comment by paper42** [comment]
yes

**Comment by Zoh-j02r** [comment]
With this change I believe the package can be successfully merged

File: srcpkgs/newm/template [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+checksum=" a03e9a4dd51b767059f9c53049597ac6e754cee11029f66b00d32031128310b4
+ 4df9bcb4c98aea1f4c452a29805d96a3066109a1424612298cc7e83f4d0ae771
+ c012482fafdad9bd4d9d75dcb5b61e28e5d60eb9f77ba0bec7c15ac2870fb70a"
+
+
+post_extract() {
+	cd $wrksrc/pywm-${version}
+	rmdir subprojects/wlroots
+	mv -v $wrksrc/wlroots-${_wlroots_commit} subprojects/wlroots
+}
```
[the remark this answers is not in the document]
Comment (Zoh-j02r) [reply srcpkgs/newm/template:28]: if not initializing the submodule newm refuses to compile, I dont know it there a better solution for the submodule init problem

File: srcpkgs/newm/template [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+
+do_build() {
+	cd $wrksrc/pywm-${version}
+	python3 setup.py build
+	cd $wrksrc/newm-${version}
+	python3 setup.py build
+}
```
[the remark this answers is not in the document]
Comment (Zoh-j02r) [reply srcpkgs/newm/template:35]: I was kinda of following the template made for Arch Linux by the creator [here](https://aur.archlinux.org/cgit/aur.git/tree/PKGBUILD?h=newm-git).

[the remark this answers is not in the document]
Comment (paper42) [reply srcpkgs/newm/template:35]: We are not AUR, can we just use two packages?

File: srcpkgs/newm/template (hunk 1 above)
[the remark this answers is not in the document]
Comment (paper42) [reply srcpkgs/newm/template:28]: Using system wlroots is a requirement for this to even be mergeable.

**Comment by Zoh-j02r** [comment]
I made this [issue](https://github.com/jbuchermn/pywm/issues/21) suggestion the addition for an option to use the system wlroots, if for some technical reason or disinterest for the developer in using a stable wlroots release, then it is unlikely the port of newm to Void, since there is low activity in the newm project for a while it might take some time. I may close this issue and reopen it if there is some progress.

File: srcpkgs/newm/template (hunk 2 above)
[the remark this answers is not in the document]
Comment (eli-schwartz) [reply srcpkgs/newm/template:35]: "What the upstream developer does in a custom recipe that has no QA or review process" is not relevant to *any* distro.

**Pull request closed without merging**